hs_communities/static/js/dual-listbox.js

```diff
@@ -0,0 +1,177 @@
+var TitleAssistantApp = new Vue({
+    el: "#title-assistant",
+    data: {
```

Comment:
@Maurier note in all of my VueJS code I'm not using a delimeter, which I see in all of your Vue classes. I also am calling jQuery with $ in my VueJS code. Let's sync up on this.

Reply:
I've been using a custom delimeter to avoid conflicts with that of Djando template language, but if your HTML doesn't have any Django delimeters you should be fine.